source/macos/AdaptiveCards/AdaptiveCards-bridge/BridgeUtils/BridgeTextUtils.mm

```diff
@@ -136,4 +137,32 @@ + (ACSRichTextElementProperties * _Nonnull)convertFactToRichTextElementPropertie
     [textProp setLanguage:[fact getLanguage]];
     return textProp;
 }
+
++ (ACSRichTextElementProperties * _Nonnull)convertInputChoiceToRichTextElementProperties:(ACSChoiceInput * _Nonnull)choiceInput
```

Comment:
both these methods have many common lines. can it be extracted to one method.

Reply:
Yes, we used direct title text because there is no any other dependancy.